docs/control-structures.rst

```diff
@@ -22,6 +22,42 @@ there is in C and JavaScript, so ``if (1) { ... }`` is *not* valid Solidity.
 
 .. _function-calls:
 
+Input Parameters and Return Parameters
+======================================
+
+Like in Javascript and in C, functions may take parameters as input;
+unlike in Javascript and C, they may also return arbitrary parameters as
+output. The input parameters, and types, must be named, type followed
+by name. For example, suppose we want our contract to accept messages
+with two integers, we would write something like::
+
+    contract Simple {
+        function(uint _a, uint _b) {
```

Comment:
The function has to have a name. Furthermore, I perhaps would not use the term "message" here.